tasks/validate.js

```diff
@@ -20,8 +20,8 @@ const config = require('./config');
 
 function validate() {
   return gulp.src([
-        `${config.dest.templates}/**/*.html`,
-        `!${config.dest.hl_partials}/**/*.html`,
+        `${config.dest.templates}/**/*amp.html`,
```

Comment:
pls dont change any existing conventions. You will need to do this in a separate PR and have it reviewed and we will need to make sure it does not break anything else.